irohad/main/impl/consensus_init.cpp

```diff
@@ -16,12 +16,12 @@
  */
 
 #include "main/impl/consensus_init.hpp"
-
```

Comment:
I'd keep this line to separate related header like (https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).